scripts/deploy-ebpf.ps1.in

```diff
@@ -6,226 +6,62 @@
 ##
 $source_directory="."
 
-# The following files should be installed on all platforms.
-[System.Collections.ArrayList]$built_runtime_files=@(
+# The following files are required for installing Debug & Release versions of eBPF for Windows.
+[System.Collections.ArrayList]$install_files = @(
+    "ebpf-for-windows.msi",
+    "setup-ebpf.ps1",
+    "vc_redist.x64.exe",
+    # VC++ Redistributable Debug Runtime DLLs.
+    "concrt140d.dll",
+    "msvcp140d.dll",
+    "msvcp140d_atomic_wait.dll",
+    "msvcp140d_codecvt_ids.dll",
+    "msvcp140_1d.dll",
+    "msvcp140_2d.dll",
+    "vccorlib140d.dll",
+    "vcruntime140d.dll",
+    "vcruntime140_1d.dll",
+    "vcruntime140_threadsd.dll",
+    "ucrtbased.dll"
+    )
+
+# The following files are only needed for testing and debugging.
+# Note: the \scripts\setup.vcxproj already copies the Debug VC++ Redist DLLs to the install directory, including the official Release installer.
+[System.Collections.ArrayList]$built_test_files = @(
+    "*.exe",
+    "*.dll",
+    "*.pdb",
+    "*.o",
+	"*.sys"
```

Comment:
Keep indentation of line 35 same as 31-34

Reply:
fixed